server/views/temporary-accommodation/components/location-header/macro.njk

```diff
@@ -4,13 +4,17 @@
 
 <div class="location-header">
   {% set premises = locationDetails.premises %}
+      {% set room = locationDetails.room %}
 
   {% if premises %}
     {% if not hideAddress %}
       <h2>Property address</h2>
-      <p>{{ premises.addressLine1 }}
-            {% if premises.town.length %}<br/>{{ premises.town }}{% endif %}
-            <br />{{ premises.postcode }}</p>
+      <p>
+            {{ room.name }}<br />
+            {{ premises.addressLine1 }}<br/>
+            {% if premises.town.length %}{{ premises.town }}{% endif %}<br />
+            {{ premises.postcode }}
```

Comment:
I think we're losing the original logic which would only add a line break where needed if the town was shown -- the same approach should be used for the `{{ room.name }}` line, I guess. Here's what it looks like with the extra blank line rendered at the start:

<img width="536" alt="Screenshot 2024-04-22 at 12 38 09" src="https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/assets/496382/a8685575-037c-4c4e-b5fd-bdf348eb4bae">



To ensure we're not adding blank lines when there is no data to show:

```suggestion
            {% if room.name.length %}{{ room.name }}<br />{% endif %}
            {{ premises.addressLine1 }}<br/>
            {% if premises.town.length %}{{ premises.town }}<br />{% endif %}
            {{ premises.postcode }}
```

Reply:
Thanks @froddd I've added the conditional line breaks back in and done the same for room.name

Reply:
Seems to have sorted it 👍